### PR TITLE
completed javascript-data-model

### PR DESCRIPTION
**Pull request by ellie-hwang**
javascript-data-model

### CODE REVIEW

File: javascript-data-model/order-history.js [hunk 1]
```diff
@@ -0,0 +1,78 @@
+var orderHistory = [
+  {
+    book: {
```
Comment (codingcodymiller) [inline javascript-data-model/order-history.js:3]: What if they order more than one book? I don't think naming the property after the type of item is the right way to go, you should store all items together in one space, and elaborate on the type of item alongside the data that describes the item itself.

File: javascript-data-model/order-history.js [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+    },
+    itemImg: true,
+    orderDate: 'July 3, 2020',
+    orderStatus:
+    {
+      deliveryStatus: true,
+      deliveryDate: 'July 5, 2020',
+      returnCloseDate: 'August 4, 2020'
+    }
+  }
```
Comment (codingcodymiller) [inline javascript-data-model/order-history.js:73]: This looks like two separate order objects for the same order. You should have one object for each order that stores the data about that order, including a property to store the data for the items in the order.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

File: javascript-data-model/order-history.js (hunk 2 above)
Comment (ellie-hwang) [reply javascript-data-model/order-history.js]: javascript-data-model-02

**Review by codingcodymiller — APPROVED** [review]
Nice work!

**Pull request merged**